pyproject.toml

```
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "ansys-dpf-post"
+version = "0.2.dev0"
```

```suggestion
version = "0.3.dev0"
```

I think the main branch should be mapping ``0.3.dev0`` at this point. There is already a ``0.2.*`` version